plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/AbstractPinotIntegrationSmokeTest.java

```diff
@@ -748,6 +764,51 @@ public void testNonLowerCaseColumnNames()
                 .isFullyPushedDown();
     }
 
+    @Test
+    public void testNonLowerTable()
+    {
+        long rowCount = (long) computeScalar("SELECT COUNT(*) FROM " + MIXED_CASE_TABLE_NAME);
+        List<String> rows = new ArrayList<>();
+        for (int i = 0; i < rowCount; i++) {
+            rows.add(format("('string_%s', '%s', '%s')", i, i, initialUpdatedAt.plusMillis(i * 1000).getEpochSecond()));
+        }
+
+        String mixedCaseColumnNamesTableValues = rows.stream().collect(joining(",", "VALUES ", ""));
```

Comment:
Also test information_schema. See BaseCaseInsensitiveMappingTest for inspiration of other useful things to test.